frontend/src/pages/GroupPage.jsx

```diff
@@ -1,15 +1,152 @@
 import Template from "../components/template";
+import ModalHeader from "../components/modalHeader.jsx";
+import LeftPanel from "../components/leftPanel";
+import Button from "../components/button";
+import { useState, React, useEffect } from "react";
+import AddMemberWindow from "../components/addMemberWindow";
+import "../css/personalPage.css";
+import UploadNotesWindow from "../components/uploadNotesWindow";
+import { useParams } from "react-router";
+import SectionTitle from "../components/sectionTitle.jsx";
 
 function GroupPage(props) {
+    const [buttonAddMember, setButtonAddMember] = useState(false);
+    const [buttonGroupProfile, setButtonGroupProfile] = useState(false);
+    const [members, setMembers] = useState([]);
+    const [groups, setGroups] = useState([]);
```

Comment:
I don't think you need groups in the group page

Reply:
Deleted it.